src/Microsoft.OpenApi.OData.Reader/Microsoft.OpenAPI.OData.Reader.csproj

```diff
@@ -23,6 +23,7 @@
     <PackageReleaseNotes>
 - Use convert setting to toggle between referencing @odata.count and @odata.nextLink #282
 - Fixes URL Path parameters of type datetime generated as strings with quotes #262
+- Set assembly version used for conversion in OpenApiInfo object #208
```

Comment:
Update the version number as well: `1.2.0`

Reply:
Preview 3 no?